README.md

```diff
@@ -64,13 +68,26 @@ Base template has been designed and developed by Themefisher. For a preview, che
 The code has been restructured, unnecessary code has been removed, comments were added wherever required, proper indentation was provided, and simple names were used which makes it a contributor-friendly codebase. So, hope on and contribute now!
 
 ## Feedback/Suggestions and Issues
+
 Feedback/Suggestions are always welcome and do mention any issues found. They can be provided through:
-1)Issues section of this repository
-2)[https://github.com/sugarlabs/www-sugarlabs/issues/63](https://github.com/sugarlabs/www-sugarlabs/issues/63)
+
+1. Issues section of this repository
+2. [https://github.com/sugarlabs/www-sugarlabs/issues/63](https://github.com/sugarlabs/www-sugarlabs/issues/63)
```

Comment:
You can change the hyperlink text to "Issue #63" and this removes the need for the hyperlink.

Reply:
@chimosky  But it closed I guess 
So could explain what i have to so now

Reply:
Thanks for pointing that out, just noticed and there seems to be two sections of `Feedback/Suggestions and Issues`, you can remove one - preferably the second -.

Reply:
@chimosky Now Check it once

Reply:
The section includes the items listed.

Reply:
Now?